### PR TITLE
Praneeth/add seconary domains

### PR DESCRIPTION
**Pull request by arepraneeth**
* Adding support for secondary domains and ipSets
* Fixed sandbox url for dnsmadeeasy

### CODE REVIEW

**Comment by kigster** [comment]
Wow thanks for the great contribution! 

Can you add a link to the readme that explains what secondary domains are? I am not familiar with the term.

File: lib/dnsmadeeasy.rb [hunk 1]
```diff
@@ -23,6 +23,9 @@ class InvalidCredentialsFormatError < Error;
   class NoSuchAccountError < Error;
   end
 
+  class NoDomainError < StandardError;
```
Comment (kigster) [inline lib/dnsmadeeasy.rb]: This should inherit from Error

**Review by kigster — CHANGES_REQUESTED** [review]
Minor neat pick, and I need to fix Travis before we merge.

**Comment by arepraneeth** [comment]
> Wow thanks for the great contribution!
> 
> Can you add a link to the readme that explains what secondary domains are? I am not familiar with the term.

Here is documentation from dnsmadeeasy about secondary dns : https://dnsmadeeasy.com/services/secondarydns/

Basically, there work DNS-Slaves, they can answers dns queries authoritatively by fetching actual zone from different DNS masters

File: lib/dnsmadeeasy.rb (hunk 1 above)
Comment (arepraneeth) [reply lib/dnsmadeeasy.rb]: Sure, changed, and rebased.

**Pull request merged**